dll/win32/shell32/shlexec.cpp

```diff
@@ -782,17 +784,56 @@ static UINT SHELL_FindExecutable(LPCWSTR lpPath, LPCWSTR lpFile, LPCWSTR lpVerb,
         return 33;
     }
 
-    if (SearchPathW(lpPath, lpFile, L".exe", ARRAY_SIZE(xlpFile), xlpFile, NULL))
+    GetCurrentDirectoryW(ARRAY_SIZE(curdir), curdir);
+    if (!PathIsFileSpecW(lpFile))
     {
-        TRACE("SearchPathW returned non-zero\n");
-        lpFile = xlpFile;
-        /* The file was found in the application-supplied default directory (or the system search path) */
+        BOOL found = FALSE;
+        if (lpPath && *lpPath)
+        {
+            TRACE("ASDF %s\n", debugstr_w(lpPath));
```

Comment:
ASDF?

Reply:
Yep, this is simply taken from Wine as-is. But if it's not necessarily to keep the code synced, this trace can be improved as well too. Feel free to suggest better format string.

Reply:
Anything is better. Nothing, `lpPath`, `Path for file`.

Reply:
OK, updated.